automation/taskcluster/release/convert_group_and_artifact_ids.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python3
+"""
+Temporary script to massage some of the group/artifact ids. Does the following:
+a. download upstreamZip files
+b. determine what artifacts should change
+c. open each zip, rename files if needed, and regenerate the pom.xml
+(+ checksums). pom.xml must be regenerated because it list dependencies,
+which might have changed (eg. if org.mozilla.components:support-base,
+every package that depends on it must be updated)
+"""
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import aiohttp
+import asyncio
+import hashlib
+import logging
+import os
+import shutil
+import sys
+import tempfile
+import zipfile
+
+from xml.etree import ElementTree
+import taskcluster.aio
+
+from scriptworker.exceptions import DownloadError
+from scriptworker.utils import download_file, retry_async, raise_future_exceptions
+
+
+log = logging.getLogger(name=__name__)
+
+SUPPORTED_HASHES = ('md5', 'sha1')
+
+ElementTree.register_namespace('', 'http://maven.apache.org/POM/4.0.0')
+
+XML_NAMESPACES = {
+    'm': 'http://maven.apache.org/POM/4.0.0',
```

Comment:
Maybe worth declaring this as constant and re-use since it's used at line`37` as well but, since this script is going away in a few weeks/months, maybe it's not worth polishing all the nits.

Reply:
Agreed